pkg/virt-launcher/virtwrap/agent-poller/agent_poller_test.go

```diff
@@ -103,19 +103,19 @@ var _ = Describe("Qemu agent poller", func() {
 			const interval = 1
 			const expectedExecutions = 3
 
-			commandExecutions := runPollAndCountCommandExecution(interval, expectedExecutions, 0, unitTestPollInitialInterval)
+			commandExecutions := runPollAndCountCommandExecution(interval, expectedExecutions, 0, pollInitialInterval)
 
 			Expect(commandExecutions).To(Equal(expectedExecutions))
 		})
 
 		It("executes the agent commands based on the minimum interval at initial run", func() {
-			const interval = 30
-			const expectedExecutions = 2
+			const interval = 3
+			const expectedExecutions = 3
```

Comment:
Please confirm it fails with this value and the 1sec initial-interval when the fix is not in place.

Reply:
Confirmed already yesterday locally